meta-xt-prod-cockpit-rcar-driver-domain/recipes-kernel/linux/files/xen-chosen.dtsi

```diff
@@ -0,0 +1,30 @@
+/*
+ * Device Tree default chosen node for systems running XEN hypervisor
+ *
+ * Copyright (C) 2019 EPAM Systems.
+ *
+ * This file is licensed under the terms of the GNU General Public License
+ * version 2.  This program is licensed "as is" without any warranty of any
+ * kind, whether express or implied.
+ */
+
+
+/ {
+	chosen {
+		bootargs = "loglvl=none guest_loglvl=none dom0_mem=256M console=dtuart dtuart=serial0 dom0_max_vcpus=4 bootscrub=0 sched_ratelimit_us=100 hmp-unsafe=true xsm=flask console_timestamps=boot";
```

Comment:
Correct me, but looks like you are creating this dtsi only for this property.
Maybe, also for `/delete-property/stdout-path`.

Why do you have `modules` block then?

Reply:
Yes, only for the property.

Reply:
Done